unityfiles/Assets/Scripts/Helper/consts.cs

```diff
@@ -45,6 +45,21 @@ public enum LEVEL_STATE {
     /// </summary>
     RELOAD,
     
+    /// <summary>
+    /// Game pause
+    /// </summary>
+    PAUSE,
+    
+    /// <summary>
+    /// Resume game
+    /// </summary>
+    RESUME,
+    
+    /// <summary>
+    /// Confirm
+    /// </summary>
+    CONFIRM,
```

Comment:
RESUME og CONFIRM bør fjernes. Knapp logikk.

Reply:
Enig ang `CONFIRM`. Men vi bør vel ha et slags `RESUME` event?

Reply:
Tenker du noe bør skje mellom Pause-> Unpause ? I så fall kan det være greit. Men utover det kan vel bare staten sette tilbake til Hunting: Som dog bør renames 😢

Reply:
For øyeblikket har jeg ikke noe behov for at det skjer noe mellom pause og unpaused. Kan godt bare sette staten tilbake til `HUNTING`, men var litt usikker på hva denne staten gjør. Om den bare er som Resume så kanskje vi bare skal rename `HUNTING` -> `RESUME`?

Reply:
Hunting er selve play mode så vidt jeg har forstått. RESUME føler jeg er en engangs state, og ikke en state man er i om det gir mening ❓ PLAY(ing) da?

Reply:
Skjønner og enig. Bytte til `LEVEL_STATE.PLAY`. Det gir jo mer mening i forhold til lesbarhet også